src/pages/frontpage/SearchSection.tsx

```diff
@@ -0,0 +1,92 @@
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { SearchTypeDropdown, SearchTypeValue } from '../search/SearchTypeDropdown';
+import { Box, Button, Typography } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { dataTestId } from '../../utils/dataTestIds';
+import { SearchTextField } from '../search/SearchTextField';
+import EastIcon from '@mui/icons-material/East';
+import { UrlPathTemplate } from '../../utils/urlPaths';
+
+const getCorrectParameters = (inputValue: string, searchType: SearchTypeValue) => {
+  switch (searchType) {
+    case SearchTypeValue.Result:
+      return inputValue ? `query=${inputValue}` : '';
+    case SearchTypeValue.Person:
+      return `type=person${inputValue ? `&name=${inputValue}` : ''}`;
+    case SearchTypeValue.Project:
+      return `type=project${inputValue ? `&multiple=${inputValue}` : ''}`;
+    default:
+      return '';
+  }
+};
+
+export const SearchSection = () => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const [selectedSearchType, setSelectedSearchType] = useState(SearchTypeValue.Result);
+  const [inputValue, setInputValue] = useState('');
+
+  const onSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    const searchTerm = getCorrectParameters(inputValue, selectedSearchType);
+    navigate({ pathname: UrlPathTemplate.Root, search: searchTerm });
+  };
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.75rem',
+        width: '100%',
+        bgcolor: '#D9D9D9',
+        borderRadius: '1rem',
+        p: '2rem 3rem',
+      }}>
+      <Box
+        sx={{ display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, gap: '0.75rem' }}
+        component="form"
+        onSubmit={onSubmit}>
+        <SearchTypeDropdown
+          selectedValue={selectedSearchType}
+          onSelectResult={() => setSelectedSearchType(SearchTypeValue.Result)}
+          onSelectPerson={() => setSelectedSearchType(SearchTypeValue.Person)}
+          onSelectProject={() => setSelectedSearchType(SearchTypeValue.Project)}
+        />
+        <SearchTextField
+          dataTestId={dataTestId.frontPage.searchInputField}
+          placeholder={t('search.search_placeholder')}
```

Comment:
SKulle hatt egen placeholder for hver type. Tekstene finnes allerede i vanlig søk, så er bare å gjenbrukje dem

<img width="482" height="47" alt="Image" src="https://github.com/user-attachments/assets/0b6b9927-d742-4099-b629-d6ae5d06b93a" />